addon/templates/components/power-calendar/days.hbs

```diff
@@ -1,26 +1,32 @@
-<div class="ember-power-calendar-row ember-power-calendar-weekdays">
+<div class="ember-power-calendar-row ember-power-calendar-weekdays ember-power-calendar--days-of-week">
   {{#each weekdaysNames as |wdn|}}
     <div class="ember-power-calendar-weekday">{{wdn}}</div>
   {{/each}}
 </div>
 
 <div class="ember-power-calendar-day-grid" onkeydown={{action "onKeyDown" calendar}}>
   {{#each weeks key='id' as |week|}}
-    <div class="ember-power-calendar-row ember-power-calendar-week" data-missing-days={{week.missingDays}}>
+    <div class="ember-power-calendar-row ember-power-calendar-week" >
       {{#each week.days key='id' as |day|}}
-        <button type="button"
-          data-date="{{day.id}}"
-          class="ember-power-calendar-day {{if onSelect 'ember-power-calendar-day--interactive'}} {{if day.isCurrentMonth 'ember-power-calendar-day--current-month' 'ember-power-calendar-day--other-month'}} {{if day.isSelected 'ember-power-calendar-day--selected'}} {{if day.isToday 'ember-power-calendar-day--today'}} {{if day.isFocused 'ember-power-calendar-day--focused'}} {{if day.isRangeStart 'ember-power-calendar-day--range-start'}} {{if day.isRangeEnd 'ember-power-calendar-day--range-end'}}"
-          onclick={{action calendar.actions.select day calendar}}
-          onfocus={{action 'onFocusDay' day}}
-          onblur={{action 'onBlurDay'}}
-          disabled={{day.isDisabled}}>
-          {{#if hasBlock}}
-            {{yield day publicAPI}}
-          {{else}}
-            {{day.number}}
-          {{/if}}
-        </button>
+        {{#if day.isVisible}}
```

Comment:
After your changes in the component you need to do this `{{if}}` for every day in the month. Why do this? It seems unrelated with the overall goal of the PR, which is add `data-day-of-week="{{day.dow}}"` to the days.